### PR TITLE
feat: configure frontend to work with both languages

### PR DESCRIPTION
**Pull request by malssid**
To switch between languages you now only have to change `BACKEND_URL` and `BACKEND_LANGUAGE` in your `.env`

### CODE REVIEW

File: app/javascript/lib/phoenix-graphql-backend.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import { hasSubscription } from '@jumpn/utils-graphql';
+import { HttpLink } from 'apollo-link-http';
+import { split } from 'apollo-link';
+import Cookies from 'js-cookie';
+import * as AbsintheSocket from '@absinthe/socket';
+import { createAbsintheSocketLink } from '@absinthe/socket-apollo-link';
+
+const webSocketURL = new URL(process.env.BACKEND_URL);
+
+webSocketURL.protocol = 'ws:';
```
Comment (cbarber) [inline app/javascript/lib/phoenix-graphql-backend.js:12]: :100: nice way to switch the protocol without find/replace

File: app/javascript/components/app.jsx [hunk 2]
```diff
@@ -2,78 +2,26 @@ import 'regenerator-runtime/runtime';
...
-
-const absintheLink = split(
-  operation => hasSubscription(operation.query),
-  webSocketLinkAbsinthe,
-  authedHttpLink,
-);
+if (process.env.BACKEND_LANGUAGE === 'ruby') {
+  link = railsLink;
+} else if (process.env.BACKEND_LANGUAGE === 'elixir') {
+  link = absintheLink;
```
Comment (cbarber) [inline app/javascript/components/app.jsx:23]: NIt: we could change these to requires for tree shaking but that can be an improvement for another day.

File: app/javascript/components/app.jsx [hunk 3]
```diff
@@ -2,78 +2,26 @@ import 'regenerator-runtime/runtime';
...
+}
```
Comment (cbarber) [inline app/javascript/components/app.jsx:24]: Should we else here and fail loudly? Or just default to `ruby` for now?

**Review by cbarber — COMMENTED** [review]
Looks great. Small comment on `if/else` on `process.env.BACKEND_LANGUAGE`

File: app/javascript/lib/phoenix-graphql-backend.js (hunk 1 above)
Comment (malssid) [reply app/javascript/lib/phoenix-graphql-backend.js:12]: Thanks!

File: app/javascript/components/app.jsx (hunk 2 above)
Comment (malssid) [reply app/javascript/components/app.jsx:23]: I was looking into that... I was trying out `import()` from here [https://javascript.info/modules-dynamic-imports](https://javascript.info/modules-dynamic-imports), but I was having trouble actually getting the default import.

File: app/javascript/components/app.jsx (hunk 3 above)
Comment (malssid) [reply app/javascript/components/app.jsx:24]: I was thinking we default to ruby for now.

**Review by cbarber — APPROVED** [review]
LGTM. Nice work :shipit:

**Pull request merged**